modules/booklismanager.js

```diff
@@ -1,21 +1,23 @@
+/* eslint-disable import/prefer-default-export */
+
```

Comment:
- I see that you have linters disabled, please I suggest that you try fixing the issues raised by linters instead of disabling them. Remember linters helps to ensure that your code is clean and easy to read 👍. Hint: In this instance, linters is expecting that you export as default since you are exporting just the class